uv.gyp

```diff
@@ -305,8 +306,9 @@
         ['OS=="zos"', {
           'sources': [
             'src/unix/pthread-fixes.c',
-            'src/unix/pthread-barrier.c'
-            'src/unix/os390.c'
```

Comment:
Is this fixing a pre-existing bug, or is a `,` not needed if there is a *NL*? It seems the original had `pthread-barrier.c'` not having a trailing `,`, it just caught my eye because the diff seemed larger than I'd expect, when you are just adding one line to the end of the array.